### DISCUSSION TITLE
Having a problem with live Forms and the Authnet hosted extension.

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1040] [Q&A]
I have set up a extensive form with Live Forms and it is working great.  https://dev.jcesba.org/join-today
I added the authnet hosted extension. set up went well. Have login ID and Transaction Key all setup and sandbox checked. When I fill out the form and submit a modal modal backdrop is a applied to the page but the modal is too far down the screen and you have to scroll down to find it.   How can I modify in order to present the Modal higher up on the screen. 

I added "#divAuthorizeNetPopup {top:0% !important}'" to the custom styles but that is really not the right way to do it IMHO

Also I am having troubles with fields from the form mapping properly  specifically the EMAIL field, the state dropdown and the phone fields are not populating into the setup for field mapping

Do I report all this stuff as Vanajaro problems or as Mandeeps problems. I have not used mandeeps in pure dnn before.

Thanks for you help.

![image](https://user-images.githubusercontent.com/6786299/113926898-62a6bf00-97b2-11eb-9774-b05f597c69c3.png)

![image](https://user-images.githubusercontent.com/6786299/113926950-7520f880-97b2-11eb-84df-29bd51b5d479.png)

**Comment by meetmandeep** [comment]
You would normally report these over at Mandeeps.com for compatibility with Vanjaro; however, Vanjaro will get its own Form Builder module that is tightly integrated with Vanjaro. We're working on a release and it will be available soon after Vanjaro is officially released.
